webapp-django/crashstats/monitoring/tests/test_views.py

```diff
@@ -178,3 +185,53 @@ def mocked_get(url, **options):
         eq_(data['status'], 'Broken')
         eq_(data['broken'], ['job1'])
         eq_(data['blocked'], ['job2', 'job3'])
+
+    @mock.patch('requests.get')
+    def test_crontabber_status_not_run_for_a_while(self, rget):
+
+        then = (
```

Comment:
I mean, I do understand the meaning of the word, but it's quite unclear because the word ``then`` can have a lot of meanings, especially in programming (I'm thinking about Promises whenever I read it, for example). I'd go for something like ``yesterday`` or ``long_ago`` instead.